scripts/release/version.js

```diff
@@ -14,6 +14,9 @@ const ARGS = [
     'version',
     // Use exact version number without using semver notation (i.e., ~ and ^)
     '--exact',
+    // Skip lerna change detection and force all the package to be re-published
```

Comment:
This flag doesn't actually publish anything so this comment is a little misleading for me.
```suggestion
    // Skip lerna change detection and force version number updates across all packages
```